react/features/chat/components/web/Chat.tsx

```diff
@@ -160,6 +172,8 @@ const Chat = ({
     t
 }: IProps) => {
     const { classes, cx } = useStyles();
+    const sortedParticipants = useSelector(getChatRecipientsOptions);
```

Comment:
getChatRecipientsOptions always returns a new object which means it will be re-rendered on every action.

Reply:
you are right, there where many unnecessary renders - I refactored so now repopulation of the drop down happens when needed only.